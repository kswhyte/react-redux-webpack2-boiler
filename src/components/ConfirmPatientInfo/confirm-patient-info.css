@import "imports/_helpers.css";

.confirm-patient-info-block {
}

.nav-tabs {
  & > li > a {
    color: var(--almost-black);
    font-weight: 700;
    letter-spacing: 1px;
  }

  & > li.active > a {
    color: var(--white);
    background-color: var(--dark-sky-blue);
    /*border: var(--dark-sky-blue);*/
  }
}

/*.nav-tabs > li.active > a:focus,
.nav-tabs > li.active > a:hover {*/

/*hr {
  color: var(--cloudy-blue);
  background-color: var(--cloudy-blue);
  text-align: center;
  height: 2px;
}*/
