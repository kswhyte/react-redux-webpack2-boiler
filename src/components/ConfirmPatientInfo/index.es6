import Component from './ConfirmPatientInfo';
export default Component;
