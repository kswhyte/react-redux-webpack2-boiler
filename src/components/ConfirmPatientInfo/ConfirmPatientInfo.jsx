import React from 'react';
// import { Route } from 'react-router-dom';

// import PatientInfo from '../PatientInfo';
// import PatientAppointments from '../PatientAppointments';
// import PatientNotes from '../PatientNotes';

import { Link } from 'react-router-dom';

import './confirm-patient-info.css';

const ConfirmPatientInfo = () => {
  return (
    <div className="confirm-patient-info-block">
      <h1>ConfirmPatientInfo</h1>
      <ul className="nav nav-tabs">
        <li className="general-info-tab active">
          <Link to="patientinfo/generalinfo">General Info</Link>
        </li>
        <li className="appointments-tab">
          <Link to="patientinfo/appoinments">PatientName Appointments</Link>
        </li>
        <li className="notes-tab">
          <Link to="patientinfo/notes">Notes</Link>
        </li>
      </ul>
    </div>
  );
};

ConfirmPatientInfo.propTypes = {};

export default ConfirmPatientInfo;
