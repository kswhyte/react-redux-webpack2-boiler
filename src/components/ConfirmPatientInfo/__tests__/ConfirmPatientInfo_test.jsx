import React from 'react';
// import { storiesOf, action, linkTo } from '@kadira/storybook';
import ConfirmPatientInfo from '../ConfirmPatientInfo';

// import { specs, describe, it } from 'storybook-addon-specifications';

import {storiesOf, describe, it, specs
} from "../../../../.storybook/facade";

import {shallow} from 'enzyme';
import expect from 'expect';


const stories = storiesOf('ConfirmPatientInfo', module);

stories.add('Story', () => {
    const confirmPatientInfoStory = (
      <ConfirmPatientInfo/>);

    specs(() => describe('Show a successful alert', () => {
        it('Should render the ConfirmPatientInfo component without crashing', () => {
            let output = shallow(confirmPatientInfoStory);
            expect(output.find('.confirm-patient-info-block').length).toEqual(1);
        });
    }));
    return confirmPatientInfoStory;
});
