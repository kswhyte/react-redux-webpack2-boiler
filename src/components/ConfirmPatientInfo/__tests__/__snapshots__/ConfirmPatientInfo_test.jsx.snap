// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Story 1`] = `
<div
  className="confirm-patient-info-wrapper"
>
  <h1>
    PatientName
  </h1>
  <div
    className="patient-tabs-wrapper"
  >
    <ul
      className="nav nav-tabs"
    >
      <li
        className="general-info-tab active"
      >
        <a
          href="/patientinfo/generalinfo"
          onClick={[Function]}
        >
          General Info
        </a>
      </li>
      <li
        className="appointments-tab"
      >
        <a
          href="/patientinfo/appointments"
          onClick={[Function]}
        >
          PatientName Appointments
        </a>
      </li>
      <li
        className="notes-tab"
      >
        <a
          href="/patientinfo/notes"
          onClick={[Function]}
        >
          Notes
        </a>
      </li>
    </ul>
  </div>
  <div
    className="footer-line"
  />
  <div>
    <div
      className="col-sm-3"
    >
      <div />
    </div>
    <div
      className="col-sm-3"
    >
      <div />
    </div>
  </div>
</div>
`;
